CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 1.1.0
+
+**CHANGES**
+
+* Allow AWS ParallelCluster Terraform provider version 1.1.0 to be supported in module examples.
```

Comment:
I'd rephrase to `Use AWS ParallelCluster Terraform provider 1.x in all module examples`.

I suggest so because when we will release provider 1.2.0, the examples will use it and this changelog entry would then be misleading